src/Service/PaymentResponseService.php

```diff
@@ -100,10 +100,6 @@ public function insertPaymentResponse(
         array $paymentResponse,
         OrderTransactionEntity $orderTransaction
     ): void {
-        if (empty($paymentResponse)) {
```

Comment:
validation for the paymentResponse data is already done before calling this method